pynetbox/core/endpoint.py

```diff
@@ -395,7 +395,7 @@ def list(self, **kwargs):
             )
         return req
 
-    def create(self, data):
+    def create(self, data={}):
```

Comment:
I know we're not modifying `data` anywhere, but I'd probably still stick to the convention of not passing in a mutable default value.

Reply:
I changed to a None default value and a `if` branch.

Reply:
Perfect, thanks!